CHANGES.md

```diff
@@ -2,8 +2,9 @@
 
 ## EODHP
 The following changes have been made for the EODHP project
-### V0.3.9 - 2024-09-04
-Bugfix to add workspace parameter to delete_collection and delete_item endpoints
+### V0.3.9 - 2024-09-06
+- Added support for the renders STAC extension at the collection level
+Bugfix to add workspace parameter to delete_collection and delete_item endpoints:
```

Comment:
Do we need to create a new release?

Reply:
This looks fine, our search-catalogue-service versions are handled elsewhere anyway